Add `make check` target for running ASL and misc tests

Index: acpica-unix-20250404/Makefile
===================================================================
--- acpica-unix-20250404.orig/Makefile
+++ acpica-unix-20250404/Makefile
@@ -20,3 +20,11 @@ else
 	include generate/unix/Makefile.config
 	include generate/unix/Makefile.common
 endif
+
+check:
+	# ASL tests
+	$(CURDIR)/tests/aslts.sh $(CURDIR)/tests/aslts $(CURDIR)/generate/unix
+
+	# misc tests
+	$(CURDIR)/debian/run-misc-tests.sh $(CURDIR) 20250404
+
